go/test/endtoend/vtgate/gen4/gen4_test.go

```diff
@@ -32,51 +32,35 @@ import (
 )
 
 func TestOrderBy(t *testing.T) {
```

Comment:
I see in this file most of the test use `utils.Exec` with `mcmp.VtConn`, could we execute the query against MySQL too?

Reply:
I tried, but it failed in a bunch of places and I didn't want go hunting it down in this PR

Reply:
got it